documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,218 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-recovery-volume_{context}"]
+= Recovering a cluster from persistent volumes
+
+This procedure describes how to recover a deleted cluster from persistent volumes.
+
+The procedure assumes that the Topic Operator is deployed.
+In situations such as these, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
+The `KafkaTopic` resources must be recovered before a cluster is started so that the topics are not deleted by the Topic Operator.
+
+If you do not use the the Topic Operator, you just need to recover the `PersistentVolumeClaim` (PVC) resources.
```

Comment:
Suggested change:

If the Topic Operator is not deployed, you need to recover the `PersistentVolumeClaim` (PVC) resources.

Reply:
If the Topic Operator is not deployed, you only need to recover the `PersistentVolumeClaim` (PVC) resources.